src/components/PopupWithForms.js

```diff
@@ -5,6 +5,8 @@ export default class PopupWithForms extends Popup {
     super({ popupSelector });
     this._popupForm = this._popupElement.querySelector(".modal__form");
     this._inputList = this._popupForm.querySelectorAll(".modal__input");
+    this._buttonElement = this._popupElement.querySelector(".modal__button");
+
     this._handleFormSubmit = handleFormSubmit;
   }
 
```

Comment:
**COULD BE IMPROVED**

You can make a special method for getting a form element from `PopupWithForm` not to **search** it in `index.js`



```js

  getForm() {
    return this._popupForm
  }


```